src/components/plugins/legacy/mapdon-ksv.js

```diff
@@ -14,7 +14,7 @@ class MapdonKSVPlugin extends BaseCommentForm {
   constructor(props) {
     super(props);
     this.pluginInstanceId = "ksv" + (0 | (Math.random() * 10000000));  // eslint-disable-line no-bitwise
-    this.state = Object.assign(this.state, {userDataChanged: false});
+    this.state = Object.assign(this.state, {userDataChanged: false, nickname: this.props.defaultNickname || ''});
```

Comment:
We actually won't need to update the legacy plugins, because the code is not to be used again in future hearings, so there's no need for the nickname functionality there. As long as the plugin renders when commenting is not enabled.